h2o-docs/src/product/data-science/algo-params/weights_column.rst

```diff
@@ -1,7 +1,7 @@
 ``weights_column``
 ------------------
 
-- Available in: GBM, DRF, Deep Learning, GLM, GAM, AutoML, XGBoost, CoxPH, Stacked Ensembles, AdaBoost
+- Available in: GBM, DRF, Deep Learning, GLM, GAM, HGLM, AutoML, XGBoost, CoxPH, Stacked Ensembles, AdaBoost
```

Comment:
Please add the following algos to the end of AdaBoost:

ANOVAGLM, ModelSelection